wremnants/theory_tools.py

```diff
@@ -176,13 +176,20 @@
 
 extended_pdf_datasets = [x for x in common.vprocs_all if not any(y in x for y in ["NNLOPS", "MiNLO"])]
 
-def expand_pdf_entries(pdf, renorm=False):
+def expand_pdf_entries(pdf, alphas=False, renorm=False):
     info = pdfMap[pdf]
-    first_entry = info.get("first_entry", 0)
-    last_entry = first_entry+info["entries"]
-    vals = [info["branch"]+f"[{i}]" for i in range(first_entry, last_entry)]
+    if alphas:
+        vals = info["alphas"]
+    else:
+        first_entry = info.get("first_entry", 0)
+        last_entry = first_entry+info["entries"]
+        vals = [info["branch"]+f"[{i}]" for i in range(first_entry, last_entry)]
+
     if renorm:
         vals = [f"std::clamp({x}/{vals[0]}*central_pdf_weight, -theory_weight_truncate, theory_weight_truncate)" for x in vals]
```

Comment:
Need std::clamp<float> to avoid float/double conflicts. Also for the line below

Reply:
Not sure I understand. What should be changed?